src/components/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div>
+      <p>Calculator</p>
+      <Display />
+      <ButtonPanel />
+    </div>
```

Comment:
You need to wrap both `Display` and `ButtonPanel` with [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax)
Reason:
We don't want to create an extra element, that's why you need to use fragment, so we are able to render without having to create an extra node wrapping `Display` and `ButtonPanel`.